terraform/deployments/govuk-publishing-platform/app_statsd.tf

```diff
@@ -1,15 +1,33 @@
+# TODO: Add Concourse deployment for statsd if/when necessary
+locals {
+  statsd_defaults = {
+    cpu                   = 1024 # TODO parameterize this
+    memory                = 2048 # TODO parameterize this
+    environment_variables = local.defaults.environment_variables
+    secrets_from_arns     = local.defaults.secrets_from_arns
+  }
+}
+
 module "statsd" {
+  aws_region                       = data.aws_region.current.name
+  backend_virtual_service_names    = [] # TODO: Add EC2 Graphite?
   cluster_id                       = aws_ecs_cluster.cluster.id
+  cpu                              = local.statsd_defaults.cpu
+  desired_count                    = var.statsd_desired_count
+  environment_variables            = local.statsd_defaults.environment_variables
   execution_role_arn               = aws_iam_role.execution.arn
-  internal_app_domain              = var.internal_app_domain
+  extra_security_groups            = [local.govuk_management_access_security_group, aws_security_group.mesh_ecs_service.id]
+  image_name                       = "statsd:test-0.1.3" # TODO: hardcoded image tag
+  log_group                        = local.log_group
+  memory                           = local.statsd_defaults.memory
   mesh_name                        = aws_appmesh_mesh.govuk.id
-  private_subnets                  = local.private_subnets
-  security_groups                  = [aws_security_group.mesh_ecs_service.id, local.govuk_management_access_sg_id]
+  registry                         = "govuk"
```

Comment:
Should this be `var.registry`?

Reply:
In this case it'll need to stay as govuk for now, since the Statsd image is not yet in ECR.